testsuite/forge-cli/src/main.rs

```diff
@@ -461,9 +461,13 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 Some(Duration::from_secs(240)),
                 None,
             )),
-        "network_latency" => config
-            .with_network_tests(vec![&NetworkLatencyTest])
-            .with_success_criteria(SuccessCriteria::new(4000, 10000, true, None, None)),
+        "three_region_simulation" => config
+            .with_initial_validator_count(NonZeroUsize::new(6).unwrap())
```

Comment:
I think we need few more validators to have enough number of validators per region? Let's make this 10 or 12?